samples/unreadVariable/out.txt

```diff
@@ -1,3 +1,6 @@
 samples\unreadVariable\bad.cpp:5:34: style: Variable 's2' is assigned a value that is never used. [unreadVariable]
     std::string s1 = "test1", s2 = "test2";
                                  ^
+samples\unreadVariable\bad.cpp:5:31: style: Variable 's2' is assigned a value that is never used. [unreadVariable]
+    std::string s1 = "test1", s2 = "test2";
+                              ^
```

Comment:
This is obviously a bug.

Reply:
I filed https://trac.cppcheck.net/ticket/13671 about it. It was also the remaining duplicated message already visible in our unit tests.